### ISSUE TITLE
code-check complains path is not a file

### ISSUE THREAD

**Issue by vaughany** [tmuras/moosh#148]
Previously code-check was working just fine:
```
$ moosh code-check -i -p path/to/code/
Registering sniffs in the moodle standard... DONE (63 sniffs registered)
Creating file list... DONE (7 files in queue)
Changing into directory path/to/code
```
...and so on.

Just before checking some code I did a git pull on the moosh repo, and when running the above code, now get the following:
```
$ moosh code-check -i -p path/to/code/
'/var/www/html/path/to/code/' is not a file
```

Running the same command with a * on the end of the path results in a `Too many argument provided (7), the maximum is: 0` error.

Running the same command but pointing to a specific file results in a PHP fatal error:
```
$ moosh code-check -i -p path/to/code/file.php
PHP Fatal error:  Uncaught exception 'UnexpectedValueException' with message 'RecursiveDirectoryIterator::__construct(path/to/code/file.php): failed to open dir: Not a directory' in /var/www/html/development/moosh/Moosh/Command/Generic/Tools/ToolsCodeCheck.php:70
Stack trace:
#0 /var/www/html/development/moosh/Moosh/Command/Generic/Tools/ToolsCodeCheck.php(70): RecursiveDirectoryIterator->__construct('path/to/code/file.php')
#1 /var/www/html/development/moosh/Moosh/Command/Generic/Tools/ToolsCodeCheck.php(47): Moosh\Command\Generic\Tools\ToolsCodeCheck->_get_files('path/to/code/file.php')
#2 /var/www/html/development/moosh/moosh.php(275): Moosh\Command\Generic\Tools\ToolsCodeCheck->execute()
#3 {main}
  thrown in /var/www/html/development/moosh/Moosh/Command/Generic/Tools/ToolsCodeCheck.php on line 70
```
I should point out that if you cd to the required directory and run ` moosh code-check -i` it works as expected.

**Comment by vaughany** [comment]
After looking through the issues and PRs, I think this would be fixed by #143.

**Comment by tmuras** [comment]
Correct - this should be fixed in #96f99cbb2203e499b4fecfab2e0287c43094d9ed
